### PR TITLE
s2n 1.0.5

### PR DESCRIPTION
**Pull request by chenrui333**
[`action-homebrew-bump-formula`](https://github.com/dawidd6/action-homebrew-bump-formula)

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/s2n.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class S2n < Formula
   desc "Implementation of the TLS/SSL protocols"
-  homepage "https://github.com/aws/s2n-tls"
-  url "https://github.com/aws/s2n-tls/archive/refs/tags/1.0.3.tar.gz"
-  sha256 "7e9783cbaa1b14bebed31f4f85019adb39222e36f59f10da86be5b1f2a3bea85"
+  homepage "https://github.com/awslabs/s2n"
+  url "https://github.com/awslabs/s2n/archive/v1.0.4.tar.gz"
+  sha256 "ca283d856043d2bef892708ea7dcd966d168b051956a4df43c9829f782b82121"
```
Comment (SMillerDev) [inline Formula/s2n.rb:5]: Source shouldn't change

**Review by SMillerDev — CHANGES_REQUESTED** [review]

File: Formula/s2n.rb (hunk 1 above)
Comment (carlocab) [reply Formula/s2n.rb:5]: Plus https://github.com/awslabs/s2n redirects to https://github.com/aws/s2n-tls anyway, so may as well skip the redirection.

**Review by SMillerDev — DISMISSED** [review]

**Comment by chenrui333** [comment]
```
ld: unknown option: -z
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [lib/libs2n.dylib] Error 1
make[1]: *** [CMakeFiles/s2n.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by carlocab** [comment]
Can we report this upstream? I've been meaning to, I just haven't found the time.

**Comment by alebcay** [comment]
Looks like previous build issue in 1.0.4 is indeed fixed in 1.0.5 as suggested in the upstream issue.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**